### DISCUSSION TITLE
[HELP] Colors doesn't work on Custom format.

### DISCUSSION THREAD

**Discussion by Ragnar-es** [fastfetch-cli/fastfetch#979] [Q&A]
I'm trying to change the color of the "Hardware Specs" phrase in the config file, but the output it's not changing.

```
{
     "type": "custom",
     "format": "┌─────── {#35} Hardware Specifications{#} ───────┐",
},
```

Output:

`┌─────── {#35}Hardware Specifications{#} ───────┐`

What am I doing wrong?

Also, how do I set the custom text bold?

**Comment by CarterLi** [comment]
`{#35}`

This is a new syntax that has not released yet

–––

Not new, but it was not supported in custom module.

**Reply by Ragnar-es** [reply]
Oh, I saw it in the example files. So I thought it was working.

Is there any way I can achieve what I'm asking?.

**Reply by CarterLi** [reply]
You should see it in a specified version of fastfetch you use. For example: https://github.com/fastfetch-cli/fastfetch/blob/2.13.1/presets/examples/2.jsonc

**Reply by CarterLi** [reply]
> how do I set the custom text bold

`\u001b[1m` is the syntax of bold text style. However some terminals disable it by default. You need to change the terminal settings if it doesn't work

<img width="237" alt="image" src="https://github.com/fastfetch-cli/fastfetch/assets/6134068/15c3a7fe-8122-4e6e-b3fa-bb8c823bef96">
